packages/backend-output-schemas/src/storage/v1.ts

```diff
@@ -1,9 +1,16 @@
 import { z } from 'zod';
 
+const bucketSchema = z.object({
+  name: z.string(),
+  bucketName: z.string(),
+  storageRegion: z.string(),
```

Comment:
Why do we need the `name` here?

Reply:
`name` needs to be included in amplify_outputs.json and schema. If we don't pass it here, we can't collect the info...or, can we?

Reply:
`name` is in the outputs. The output looks like this

```
...
  "storage": {
    "aws_region": "us-east-1",
    "bucket_name": "amplify-viteproject-zhamu-amplifyteamdrivebucket28-nriywgwjhjnt",
    "buckets": [
      {
        "name": "amplifyTeamDrive",
        "bucket_name": "amplify-viteproject-zhamu-amplifyteamdrivebucket28-nriywgwjhjnt",
        "aws_region": "us-east-1"
      },
      {
        "name": "amplifyPoCTwo",
        "bucket_name": "amplify-viteproject-zhamu-amplifypoctwobucket0466f-caqijvcd79nc",
        "aws_region": "us-east-1"
      }
    ]
  },
  ...
```